workflow/util/util_test.go

```diff
@@ -279,91 +276,26 @@ status:
   startedAt: "2020-04-10T15:21:23Z"
 `
 
-func TestResumeWorkflowCompressed(t *testing.T) {
-	wfIf := fakeClientset.NewSimpleClientset().ArgoprojV1alpha1().Workflows("")
-	origWf := unmarshalWF(suspendedWf)
-
-	clearFunc := packer.SetMaxWorkflowSize(1156)
-	defer clearFunc()
-	err := packer.CompressWorkflowIfNeeded(origWf)
-	assert.NoError(t, err)
-
-	_, err = wfIf.Create(origWf)
-	assert.NoError(t, err)
-
-	err = ResumeWorkflow(wfIf, sqldb.ExplosiveOffloadNodeStatusRepo, "suspend", "")
-	assert.NoError(t, err)
-
-	wf, err := wfIf.Get("suspend", metav1.GetOptions{})
-	assert.NoError(t, err)
-	assert.NotEmpty(t, wf.Status.CompressedNodes)
-}
-
-func TestResumeWorkflowOffloaded(t *testing.T) {
-	wfIf := fakeClientset.NewSimpleClientset().ArgoprojV1alpha1().Workflows("")
-	origWf := unmarshalWF(suspendedWf)
-
-	//set threshold so this workflow is too big for compression to be valid
-	clearFunc := packer.SetMaxWorkflowSize(10)
-	defer clearFunc()
-
-	origNodes := origWf.Status.Nodes
-
-	origWf.Status.Nodes = nil
-	origWf.Status.OffloadNodeStatusVersion = "123"
-
-	_, err := wfIf.Create(origWf)
-	assert.NoError(t, err)
-
-	offloadNodeStatusRepo := &mocks.OffloadNodeStatusRepo{}
-	offloadNodeStatusRepo.On("IsEnabled", mock.Anything).Return(true)
-	offloadNodeStatusRepo.On("Get", "4f08d325-dc5a-43a3-9986-259e259e6ea3", "123").Return(origNodes, nil)
-	offloadNodeStatusRepo.On("Save", "4f08d325-dc5a-43a3-9986-259e259e6ea3", mock.Anything, mock.Anything).Return("1234", nil)
-
-	err = ResumeWorkflow(wfIf, offloadNodeStatusRepo, "suspend", "")
-	assert.NoError(t, err)
-
-	wf, err := wfIf.Get("suspend", metav1.GetOptions{})
-	assert.NoError(t, err)
-	assert.Equal(t, "1234", wf.Status.OffloadNodeStatusVersion)
-}
-
-//workflow is too big but offload is disabled
-func TestResumeWorkflowOffloadDisabled(t *testing.T) {
```

Comment:
these tests are all irrelevant now - they were testing the hydration code by proxy - we don't need that - or want that